mobilitypath_visualizer/src/mobilitypath_visualizer.cpp

```diff
@@ -134,11 +134,6 @@ namespace mobilitypath_visualizer {
 
         auto curr_location_msg = msg; //variable to update on each iteration as offsets are measured since last traj point
         
-        //location message standard is specified in meters, while the logic here expects cm to work with offsets
```

Comment:
Same as above, one or the other should be updated if this is no longer needed.

Reply:
Same as above